AWSPinpointTests/AWSPinpointEventRecorderTests.m

```diff
@@ -305,6 +305,83 @@ - (void) testUpdateSessionCampaignAttributes {
     }];
 }
 
+- (void) testUpdateSessionOverwriteCampaignAttributesWithJourney {
```

Comment:
seems like we should have an additional test which mirrors testUpdateSessionCampaignAttributes, but is only for a journey

Reply:
This is not necessarily needed since from the AWSPinpointEvent and AWSPinpointEventRecorder point of view they are just attributes and do not need to know what the key or value is, but I will add one for journey to show that these are the possible "feature" attributes that `updateSessionStartWithFeatureAttributes` can be invoked with.